README.md

```diff
@@ -67,4 +67,4 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 
 ### `npm run build` fails to minify
```

Comment:
- [ ] According to the project requirements, we are to maintain a descriptive README file following the given [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide